extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -60,7 +68,7 @@ function render_block( $attributes ) {
 	// Sanitize the URL.
 	$attributes['url'] = esc_url_raw( $attributes['url'] );
 
-	$track_list = get_track_list( $attributes['url'], 10 );
+	$track_list = get_track_list( $attributes['url'], $attributes['itemsToShow'] );
```

Comment:
I would ensure we always pass a proper integer here.

```suggestion
	$track_list = get_track_list( $attributes['url'], absint( $attributes['itemsToShow'] ) );
```

Reply:
Let's just generally make a rule not to trust attributes at all.